frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -346,6 +382,36 @@ class ConcatenateWith {
       bridge.handleSomeStaticStreamSinkSingleArgStaticMethodConcatenateWith(hint: hint);
 }
 
+@freezed
+class CustomError with _$CustomError {
```

Comment:
What about letting CustomError `implements Exception` in Dart side. B/c people often `on Exception catch(e)`